contributing/code/bc.rst

```diff
@@ -258,6 +258,14 @@ Make public or protected                                                  Yes
 Remove private property                                                   Yes
 **Constructors**
 Add constructor without mandatory arguments                               Yes             :ref:`[1] <note-1>`
+:ref:`Add argument without a default value <add-argument-public-method>`  No
+Add argument with a default value                                         Yes
+Remove argument                                                           No              :ref:`[3] <note-3>`
+Add default value to an argument                                          Yes
+Remove default value of an argument                                       No
+Add type hint to an argument                                              No
```

Comment:
we should probably update the whole table (maybe in a separate PR) to stop talking about `type hint`. Those are not hints at all. the right term for that is `type`, or maybe `native type` if we want to compare it to phpdoc-declared types.

Reply:
I agree, I'll have a loot at it once this one is merged